### ISSUE TITLE
Gauntlet DL's incorect in HLE mode

### ISSUE THREAD

**Issue by theboy181** [gonetz/GLideN64#1803]
When you are in HLE mode the DL's only get to 60. When I load a save state created with AL+ in HLE mode the health bar is broken, but when the same AL save state in LLE mode the health bar works. 

Could the missing DL's in HLE mode be the issue with the non working health bar?

@gonetz if you need me to explain further I can show you over video on Discord64

**Comment by loganmc10** [comment]
Does the health bar work in GLideN64 LLE?

**Comment by theboy181** [comment]
yes if you load a save state.. so part of the issue might be the missing DL's

**Comment by gonetz** [comment]
> if you need me to explain further I can show you over video on Discord64

may be later.

**Comment by gonetz** [comment]
Is not it duplicate of #1647?

**Comment by theboy181** [comment]
I’m not sure that the DLs being incorrect was identified. I figure it could be linked but is a separate issue.

**Comment by oddMLan** [comment]
Duplicate of #1647 , but this ticket has more info. I suggest appending the info of this ticket into the other one and closing this

**Comment by gonetz** [comment]
Lets keep both for now. I don't know what causes that bug.

**Comment by Papermanzero** [comment]
Also fixed with recent builds.
